docs/rules/no-duplicate-imports.md

```diff
@@ -33,6 +35,16 @@ import { merge, find } from 'module';
 import something from 'another-module';
 ```
 
+Example of **correct** code for this rule (namespace imports can't be merged):
+
+```js
+/*eslint no-duplicate-imports: "error"*/
+
+import * as Namespace from 'module';
+import { merge } from 'module';
```

Comment:
This behaviour doesn't fit the [heuristic](https://github.com/eslint/eslint/issues/12758#issuecomment-572796068):

> an import that can be merged with another is a duplicate of that other.

Namespace imports and named imports can be merged with default imports.

Reply:
Thanks for your feedback @G-Rath 
I'll rework on it to fit this scenario